calib_meas_number: 4
calib_meas:
  pose1:
    force_avg:
      - 0.1521934294330592
      - 0.04844507382844252
      - 0.006304600412419553
    torque_avg:
      - -5.635950990290948e-05
      - 0.0006962514245404868
      - 8.068064758058578e-10
    gravity:
      - -9.806649064205548
      - -0.001074826044471037
      - 0.004147139431339473
  pose3:
    gravity:
      - -3.819371657673907
      - 0.01484989664177515
      - -9.032306673470906
    torque_avg:
      - 0.0005761121246524151
      - -0.0009301992653528642
      - 1.174364091457587e-09
    force_avg:
      - -0.07540153437470619
      - 0.1406359873112517
      - 0.06706553139931667
  pose2:
    force_avg:
      - -0.05367795006809145
      - 0.1390910760892196
      - 0.1147201923215103
    gravity:
      - -7.036608223269595
      - 0.01092343614638948
      - -6.830549729945423
    torque_avg:
      - 0.001152721264360326
      - -0.0009670574213896801
      - -4.417889570482863e-08
  pose0:
    gravity:
      - -7.212658566379069
      - -5.208985211634143
      - -4.124853172212982
    force_avg:
      - 0.07576230852691779
      - 0.09179926084696172
      - 0.05619690902839117
    torque_avg:
      - -1.350657547028525e-05
      - -0.001701637664215409
      - -1.980222698922983e-08